modules/vulnerabilities/unix/local/writeable_shadow/secgen_metadata.xml

```diff
@@ -23,9 +23,4 @@
   <hint>An access control misconfiguration</hint>
   <solution>Edit the shadow file to set a password for root</solution>
 
-  <conflict>
```

Comment:
Not sure why this has been removed. It shouldn't select this vuln twice for a system